.travis.yml

```diff
@@ -26,7 +26,7 @@ before_install:
 script:
   - mkdir build
   - cd build
-  - cmake -DCMAKE_VERBOSE_MAKEFILE=ON -DWITH_SYSTEMD=0 -DWITH_TESTS=1 .. && make && make test
+  - cmake -DCMAKE_VERBOSE_MAKEFILE=ON -DWITH_SYSTEMD=0 -DWITH_TESTS=1 .. && make && make test ARGS=-V
```

Comment:
This add verbose output of unit tests.  It is helpful when the test fails to know the exact failure.